CHANGES.md

```diff
@@ -100,7 +104,7 @@ OpenSSL 3.0
    read or write an EVP_PKEY directly using the OSSL_DECODER and OSSL_ENCODER
    APIs. Or load an EVP_PKEY directly from EC data using EVP_PKEY_fromdata().
 
-   *Shane Lontis and Paul Dale*
+   *Shane Lontis, Paul Dale, Richard Lewitte, and Tomas Mraz*
```

Comment:
It's been long since I saw a misspelling of my name 😉

Reply:
Heh, I'm sorry for that. :grin:

Reply:
Not a biggie, it was a bit amusing